src/test/java/seedu/address/logic/parser/SetSellerPriorityCommandParserTest.java

```diff
@@ -65,34 +66,103 @@ public void parse_invalidPriority_exceptionThrown() {
+        assertParseFailure(parser, "1 aedfaf",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, SetSellerPriorityCommand.MESSAGE_USAGE));
     }
 
     @Test
     public void parse_unrecommendedPriority_warningGiven() {
         CommandWarnings warnings = new CommandWarnings();
         warnings.addWarning(Priority.MESSAGE_RECOMMENDATIONS);
-        assertParseSuccess(parser, "1 no",
-                new SetSellerPriorityCommand(Index.fromZeroBased(0), DEFAULT_PRIORITY, warnings));
+
         assertParseSuccess(parser, "1987 hkrhjfek",
                 new SetSellerPriorityCommand(Index.fromZeroBased(1986), HIGH_PRIORITY, warnings));
-        assertParseSuccess(parser, "6 modium",
-                new SetSellerPriorityCommand(Index.fromZeroBased(5), MEDIUM_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 highlow",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, HIGH_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 hlow",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, HIGH_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 highh",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, HIGH_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 hhhhh",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, HIGH_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 hmln",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, HIGH_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 mil",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, MEDIUM_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 mfdifdsjifads",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, MEDIUM_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 mediums",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, MEDIUM_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 mhln",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, MEDIUM_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 lpw",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, LOW_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 lpwfgadadfewf",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, LOW_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 lhmn",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, LOW_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 no",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, DEFAULT_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 nedium",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, DEFAULT_PRIORITY, warnings));
+        assertParseSuccess(parser, "1 nhml",
+                new SetSellerPriorityCommand(INDEX_FIRST_PERSON, DEFAULT_PRIORITY, warnings));
+
+        // PRIORITY PARSING ISSUES:
```

Comment:
Noted.

Reply:
Thanks for the comments and looking through! Will add more test cases to come in the next PR